desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -210,6 +211,10 @@ private IconUtils() {
         // prevent instantiation
     }
 
+    public static ImageIcon getIcon(String relativePath) {
```

Comment:
I can't find any places in the code where this method is used - and it seems to be overlapping in functionality with ```ImageManager.get().getImageIcon(...)```